.github/ISSUE_TEMPLATE/documentation-edit.md

```diff
@@ -0,0 +1,13 @@
+---
+name: "\U0001F4DD Documentation Fix"
+about: Fixing a problem in an existing docs page
```

Comment:
I don't want to encourage folks to create issues for docs because it's basically writing words to remind you to write other words. It's a lot of wasted effort. PRs are the appropriate form, IMHO.

Reply:
Mmm... I get what you're saying, but I also am assuming there's going to be a bunch of discussion on how to tackle stuff for the docs rewrite.  I'm also trying to use this template for my own benefit as I create issues for the specific pages that need to be reworked.

That said, if you want to remove the docs edit template, sure.

Reply:
Yes, but that's a special case that definitely does warrant having the issues. I'm talking more about there being grammar or linking issues. An issue is just clutter for that kind of stuff.